gpiocontrol/source/android/app/src/main/java/org/kaaproject/kaa/examples/gpiocontol/utils/KaaProvider.java

```diff
@@ -25,8 +28,6 @@
 import org.kaaproject.kaa.common.endpoint.gen.UserAttachResponse;
 import org.kaaproject.kaa.examples.gpiocontrol.DeviceInfoRequest;
 import org.kaaproject.kaa.examples.gpiocontrol.RemoteControlECF;
-import android.content.Context;
-import android.util.Log;
 
 public class KaaProvider {
 
```

Comment:
I think this class must be named KaaManager like in other apps and must be in directory kaa/ like in other apps.